src/concrete_minkowski_difference.jl

```diff
@@ -0,0 +1,40 @@
+export minkowski_difference, pontryagin_difference
+
+"""
+    minkowski_difference(Q::LazySet{N}, P::LazySet{N}) where {N<:Real}
+
+Concrete Minkowski difference (geometric difference) for a pair of
+lazy sets.
+
+### Input
+
+- `Q`         -- lazy set
```

Comment:
```suggestion
- `Q` -- polytopic set
```

Reply:
Actually, Theorem 2.3 works for unbounded polyhedra. Why do you restrict this implementation to bounded (polytopic) `Q`?

Reply:
..  it requires some special handling with the infinities of the support function, i've proposed to defer that to a follow-up PR (see last comment here https://github.com/JuliaReach/LazySets.jl/issues/1478#issuecomment-513608587)

Reply:
I see. Then when this PR gets closed, we should re-read #1478 and open a follow-up issue.